vespajlib/src/test/java/com/yahoo/io/reader/NamedReaderTestCase.java

```diff
@@ -4,10 +4,11 @@
 import java.io.IOException;
 import java.io.Reader;
 import java.io.StringReader;
+import java.lang.reflect.Method;
```

Comment:
Dangling imports?

Reply:
Yes, but that's the least serious problem with this code..